### ISSUE TITLE
CELA P1_A11y_code.visualstudio.com_VisualStudioCode_Visual Studio Code Tips and Tricks (Customization)_ColorContrast: Color Contrast ratio for the text present on the page is below 4.5:1 ratio.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/vscode-docs#3601]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#CELA P1_A11y_code.visualstudio.com;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#ChromiumEdge;#DesktopWeb;#MAS1.4.3;#Win10;#Visual Studio Code;#CELAP1_code;

**Environment Details:**
Application Name: code.visualstudio.com
#URL:https://code.visualstudio.com/docs/python/python-tutorial
Windows Version: Windows10
Browser version: Microsoft Edge Version 84.0.488.1 (Official build) dev (64-bit).

**Repro Steps:**
1)Launch the application url :
code.visualstudio.com/docs/getstarted/tips-and-tricks 
2)Visual Studio Code Tips and Tricks Page will get open.
3)Go through "Customization" heading present on the page till the "Tune your settings" heading.
4)Launch accessibility insights for Desktop and check for color contrast for the text present on the screen.

**Actual:**
The color contrast of the numerical text present on the screen is below 4.5:1 ratio.

**Similar issue has been observed in following scenarios:**
1. Editing Hacks
2. IntelliSense

**Expected:**
The expected color contrast ratio of the textual content should be equal to or greater than 4.5:1 ratio to be clearly visible for low vision users.

**User Impact:**
Low vision users might not be able to clearly see all the text that is present on the screen.

**Attachment link for Reference:**
[3601_CELA P1_A11y_code.visualstudio.com_VisualStudioCode_Visual Studio Code Tips and Tricks (Customization)_ColorContrast.zip](https://github.com/microsoft/vscode-docs/files/4522207/3601_CELA.P1_A11y_code.visualstudio.com_VisualStudioCode_Visual.Studio.Code.Tips.and.Tricks.Customization._ColorContrast.zip)

**Comment by gregvanl** [comment]
Couldn't open the .zip file.
Duplicate of https://github.com/microsoft/vscode-docs/issues/3583 where the color contrast for constants within code blocks is too low (but very close 4.486:1).

**Comment by ManishaC1** [comment]
Verified the issue in below environment
URL:https://code.visualstudio.com/docs/getstarted/tips-and-tricks
Issue is fixed.
Github tags:#Closed

![Screenshot (426)](https://user-images.githubusercontent.com/105198219/216913422-f6168250-dd3a-481a-9c6b-267144609114.png)
